Client.xcodeproj/project.pbxproj

```diff
@@ -4391,7 +4394,7 @@
 					};
 					3BFE4B061D342FB800DDF53F = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = 43AQ936H96;
+						DevelopmentTeam = 9G8J6YA743;
```

Comment:
Please don't change the `DevelopmentTeam`.

Reply:
Strange, didn't realize that changed, will revert.